src/components/ha-sidebar.ts

```diff
@@ -395,18 +425,52 @@ class HaSidebar extends LitElement {
             <mwc-icon-button class="show-panel">
               <ha-svg-icon .path=${mdiPlus}></ha-svg-icon>
             </mwc-icon-button>
-          </paper-icon-item>`;
+          </ha-clickable-list-item>`;
         })}
         ${this._renderSpacer()}`
       : ""}`;
   }
 
-  private _renderDivider() {
-    return html`<div class="divider"></div>`;
+  private _renderUtilityPanels() {
+    const [, afterSpacer] = computePanels(
+      this.hass.panels,
+      this.hass.defaultPanel,
+      this._panelOrder,
+      this._hiddenPanels
+    );
+
+    // prettier-ignore
+    return html`
+      <mwc-list
+        @focusin=${this._listboxFocusIn}
+        @focusout=${this._listboxFocusOut}
+        @scroll=${this._listboxScroll}
+        @keydown=${this._listboxKeydown}
+        class="utility-panels"
+      >
+        ${afterSpacer.map((panel) =>
+          this._renderPanel(
+            panel.url_path,
+            panel.url_path === this.hass.defaultPanel
+              ? panel.title || this.hass.localize("panel.states")
+              : this.hass.localize(`panel.${panel.title}`) || panel.title,
+            panel.icon,
+            panel.url_path === this.hass.defaultPanel && !panel.icon
```

Comment:
Same here, `panel.component_name === "lovelace"`

Reply:
It looks like `this.hass.defaultPanel` is set to the constant "lovelace" already here: https://github.com/home-assistant/frontend/blob/dev/src/fake_data/provide_hass.ts#L200

Should I still use an explicit string here or should I do `panel.component_name === this.hass.defaultPanel`